h2o-core/src/main/resources/www/login.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
@micahstubbs please take a look

@mmalohlava right now everything is protected (as it was before this change - static content is served by RequestServer that requires authentication)

The login page would ideally use the unprotected resources - which would complicate things a little bit. I wanted to keep things simple and have login.html as the only unprotected resource.